tests/ihontaryk/homework_8/test_restaurant.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+
+from ihontaryk.homework_8.restaurant import ItalianRestaurant, SpanishRestaurant, \
+    AustrianRestaurant, Order
+
+menu1 = ItalianRestaurant()
+menu2 = SpanishRestaurant()
+menu3 = AustrianRestaurant()
```

Comment:
do not create objects to be tested outside the tests, since they are created during tests initialisation but not during tests run

Reply:
same relates to parametrised actually